test/e2e/apimachinery/aggregator.go

```diff
@@ -745,6 +745,10 @@ func TestSampleAPIServer(ctx context.Context, f *framework.Framework, aggrclient
 			AbsPath("/").
 			SetHeader("Accept", "application/json").DoRaw(ctx)
 		if err != nil {
+			if apierrors.IsForbidden(err) {
+				framework.Logf("Skip checking the removal of the group path due to lack of permission of reading the root path")
```

Comment:
conformance tests may not skip.
the whole point of conformance tests is to be required of all clusters.

/hold

Reply:
agreed we should not skip. I opened https://github.com/kubernetes/kubernetes/pull/122129 as an alternative that reverts the conformance change and validates the bugfix in an integration test instead